app/validators/efl_year_validator.rb

```diff
@@ -0,0 +1,27 @@
+class EflYearValidator < ActiveModel::EachValidator
```

Comment:
Should this be made more generic, so that we can use it in other places where we ask for a year (eg degree start and end years)?

```suggestion
class YearValidator < ActiveModel::EachValidator
```